includes/blocks/header/render.php

```diff
@@ -79,37 +81,57 @@ function benenson_render_header_block( array $attributes = [] ) {
 				esc_url( wp_get_attachment_url( $attributes['featuredVideoId'] ) )
 			);
 		}
-
-		print '<div class="container"><div class="hero-content">';
```

Comment:
Is there a reason why you've replaced `print`s with breaking out of PHP? I realise that `print` isn't always nice to see, but we've now got an extra three layers of indentation as a consequence.